jsonld/projects/C01-verified-carbon-standard-project.json

```
@@ -0,0 +1,56 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "qudt": "http://qudt.org/schema/qudt",
+    "unit": "http://qudt.org/2.1/vocab/unit",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@type": "regen:Project",
```

Do we want a project name? @S4mmyb 
```suggestion
  "@type": "regen:Project",
  "schema:name": ""
```

---

We have a note in the code to show the on-chain projectId if there is no name. I could just do that.

---

I think it would make sense to add the project name indeed
although we'll only be able to add the on-chain project ID once we have projects on-chain (probably not before a few months)

---

I made this addition via fb5be86b49ff521f75a1aa48a610c5a70e0d55ca